Index: sionlib/src/utils/Makefile
===================================================================
--- sionlib.orig/src/utils/Makefile
+++ sionlib/src/utils/Makefile
@@ -76,9 +76,8 @@ sionversion: sionversion.o ../lib/lib$(S
 
 ###### sionconfig ######
 install-config: sionconfig.in
-	AINSTDIR=`(cd $(PREFIX) && pwd)`; \
 	sed -e "s!@ARCH@!$(ARCH)!" \
-		-e "s!@INSTDIR@!$$AINSTDIR!" \
+		-e "s!@INSTDIR@!$(PREFIX)!" \
 		-e "s!@LIBNAME@!$(SION_LIBNAME_PAR)!" \
 		-e "s!@LIBSERNAME@!$(SION_LIBNAME_SER)!" \
 		-e "s!@HINTSLIB@!$(HINTSLIB)!" \
@@ -89,8 +88,8 @@ install-config: sionconfig.in
 		-e "s!@SIONFWD_LIBS@!$(SIONFWD_LIBS)!" \
 		-e "s!@IME_LIBS@!$(IMELIB_LIBPATH) $(IMELIB_LIB)!" \
 		-e "s!@IME_CFLAGS@!-D_SION_IME_NATIVE!" \
-		sionconfig.in > $(PREFIX)/bin/sionconfig
-	chmod 755 $(PREFIX)/bin/sionconfig; \
+		sionconfig.in > $(BINDIR)/sionconfig
+	chmod 755 $(BINDIR)/sionconfig; \
 
 
 install: all
